internal/migrate/drone/export.go

```diff
@@ -39,7 +40,7 @@ type Exporter struct {
 	Tracer tracer.Tracer
 }
 
-func (m *Exporter) Export(ctx context.Context) (*types.Org, error) {
+func (m *Exporter) Export(ctx context.Context, downgrade bool) (*types.Org, error) {
```

Comment:
for consistency, I would put downgrade in the struct instead of passing it as a method param

```diff
type Exporter struct {
	Repository repo.Repository
	Namespace  string
+	Downgrade bool

	ScmClient *scm.Client
	ScmLogin  string

	Tracer tracer.Tracer
}
```

Reply:
Thanks, fixed